mne/channels/channels.py

```diff
@@ -1016,8 +1016,8 @@ def read_ch_connectivity(fname, picks=None):
     """Parse FieldTrip neighbors .mat file.
 
     More information on these neighbor definitions can be found on the related
-    `FieldTrip documentation pages
-    <http://www.fieldtrip.org/template/neighbours>`__.
+    FieldTrip documentation pages:
+    http://www.fieldtriptoolbox.org/template/neighbours/
```

Comment:
by doing this you make the link not clickable anymore. See how it looks now:

https://martinos.org/mne/stable/generated/mne.channels.read_ch_connectivity.html